fastai2/callback/fp16.py

```diff
@@ -89,7 +89,12 @@ def begin_fit(self):
         if self.dynamic: self.count = 0
 
     def begin_batch(self): self.learn.xb = to_half(self.xb)
-    def after_pred(self):  self.learn.pred = self.pred.float()
+
+    def after_pred(self):
+        if isinstance(self.learn.pred, list):
```

Comment:
There is the to_float method to do this normally.

Reply:
Thanks, I will update it